src/extension.ts

```diff
@@ -1,19 +1,28 @@
-import { readFile } from 'fs';
 import { join } from 'path';
-import { Diagnostic, DiagnosticCollection, DiagnosticSeverity, ExtensionContext, FileSystemWatcher, languages, Position, Range, Uri, workspace, window } from 'vscode';
-import { Coverage, CoverageCollection, CoverageInfoCollection, BranchCoverageInfo, FunctionCoverageInfo, LineCoverageInfo } from './coverage-info';
+import { Diagnostic, DiagnosticSeverity, ExtensionContext, languages, Position, Range, Uri, RelativePattern, workspace, window } from 'vscode';
+import { Coverage, CoverageCollection, BranchCoverageInfo, FunctionCoverageInfo, LineCoverageInfo } from './coverage-info';
 import { parse as parseLcov } from './parse-lcov';
-import { parse as parseConverageJson } from './parse-coverage-json';
 
 export function activate(context: ExtensionContext) {
-  const searchCriteria = 'coverage/lcov*.info';
   const packageInfo = require(join(context.extensionPath, 'package.json'));
   const diagnostics = languages.createDiagnosticCollection('coverage');
   const statusBar = window.createStatusBarItem();
-  const watcher = workspace.createFileSystemWatcher(searchCriteria);
   const coverageByfile = new Map<string, Coverage>();
 
-  context.subscriptions.push(diagnostics, statusBar, watcher);
+  let searchCriteria = 'coverage/lcov*.info';

```

Comment:
🔕 maybe now is the time to create a global `const` for this?

```
const DEFAULT_SEARCH_CRITERIA = 'coverage/lcov*.info';
....
let searchCriteria = DEFAULT_SEARCH_CRITERIA;
```

Reply:
It's not used anywhere else, so I didn't consider it.

I thought about adding a const for `"searchCriteria"`, which would make some sense, since it's used three times in a row.